### PR TITLE
Feature/10411 qr code scanner/10638 info screen

### PR DESCRIPTION
**Pull request by nickguendling**
## Description
Adds an info screen to the universal QR code scanner.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10411

## Screenshots
Simulator Screen Shot - iPhone 13 Pro - 2021-11-22 at 18 47 57
Simulator Screen Shot - iPhone 13 Pro - 2021-11-22 at 18 48 01
Simulator Screen Shot - iPhone 13 Pro - 2021-11-22 at 18 48 06
Simulator Screen Shot - iPhone 13 Pro - 2021-11-22 at 18 48 08

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Comment by jucktnich** [comment]
IMO Please rename "Überprüfung von Zertifikaten bei Ticketbuchungen". This may be misunderstood as if cwa would be like CovPassCheck

**Comment by KathrinLuecke** [comment]
@jucktnich Do you think "Nachweis Ihrer Zertifikate bei Ticketbuchungen" would be better?

**Comment by jucktnich** [comment]
@KathrinLuecke yes, sounds good to me

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -3176,7 +3176,7 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "UniversalQRScanner_InstructionTitle" = "Welche QR-Codes können Sie scannen?";
 
-"UniversalQRScanner_InstructionDescription" = "Ihre PCR-Tests, Schnelltests, Testzertifikate, Impfzertifikate, Genesenenzertifikate sowie Check-ins";
+"UniversalQRScanner_InstructionDescription" = "Tests, Zertifikate, Check-ins und Überprüfungen bei Ticketbuchungen";
```
Comment (KathrinLuecke) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:3179]: Please replace "Überprüfungen" by "Nachweis"

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -3206,6 +3206,22 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
...
 
+"UniversalQRScanner_Info_title" = "Informationen zu QR-Codes";
+
+"UniversalQRScanner_Info_bulletPoint1" = "Ihre Testzertifikate, Impfzertifikate, Gensenenzertifikate, sowie Zertifikate Ihrer Familienmitglieder";
+
+"UniversalQRScanner_Info_bulletPoint2" = "Check-ins für Veranstaltungen und Orte";
+
+"UniversalQRScanner_Info_bulletPoint3" = "PCR-Tests und Schnelltests";
+
+"UniversalQRScanner_Info_bulletPoint4" = "Überprüfung von Zertifikaten bei Ticketbuchungen";
```
Comment (KathrinLuecke) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:3217]: Please replace "Überprüfung" by "Nachweis"

**Review by KathrinLuecke — CHANGES_REQUESTED** [review]
Please make the requested text changes

File: src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewController.swift [hunk 3]
```diff
@@ -30,12 +30,34 @@ class FirstTicketValidationConsentViewController: DynamicTableViewController, Fo
...
+		if let dismissHandlingNC = navigationController as? DismissHandlingNavigationController {
+			dismissHandlingNC.restoreOriginalNavigationBar()
+		}
+	}
+
+	// MARK: - DismissHandling
+
+	func wasAttemptedToBeDismissed() {
+		onDismiss()
 	}
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewController.swift:61]: Thank you!

File: src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift [hunk 4]
```diff
@@ -23,13 +23,14 @@ struct FirstTicketValidationConsentViewModel {
...
-				),
-				cells: []
+				cells: [
+					.headlineWithImage(
+						headerText: AppStrings.TicketValidation.FirstConsent.title,
+						image: UIImage(imageLiteralResourceName: "Illu_TicketValidation"),
+						accessibilityLabel: AppStrings.TicketValidation.FirstConsent.imageDescription,
+						accessibilityIdentifier: AccessibilityIdentifiers.TicketValidation.FirstConsent.image
+					)
+				]
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift:33]: what makes the difference here?

File: src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift [hunk 5]
```diff
@@ -65,7 +66,7 @@ struct FirstTicketValidationConsentViewModel {
 			// Bulletpoints in consent box
 			.section(
 				cells: [
-					.legalExtendedDataDonation(
+					.acknowledgement(
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift:69]: did not know this cell type 😅

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift (hunk 4 above)
Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift:33]: This type contains the gradient as a separate background view and pushes the illustration down if the text size of the header text is increased. The downside with this solution is, you need the illustration without the gradient in the correct size, I had to manually resize them and export them separately. But in the other solution they could overlap, that's why we invented this one when UX introduced this header style with the gradient in the background.

File: src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift (hunk 5 above)
Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/TicketValidation/FirstConsent/FirstTicketValidationConsentViewModel.swift:69]: I think it's the older one. It only supports one set of bullet points, which is enough in this case. With `legalExtendedDataDonation` there was some extra space at the bottom that we didn't need, that why I switched them.

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Review by ArturFriesen — APPROVED** [review]

**Pull request merged**